### PR TITLE
Msg enum update

### PR DESCRIPTION
**Pull request by epmikida**
These drifted out of date, and I've added a new one for charm group broadcasts, so I updated the rest of the list as well.

### CODE REVIEW

**Comment by epmikida** [comment]
I do wonder though if this could be done without adding a new ck message type. From looking at the envelope though, nothing in it would signify that the message is a broadcast so adding a message type was the most straightforward way to do this. Open to other suggestions though.

File: charm/debug/pdata/MsgInfo.java [hunk 1]
```diff
@@ -40,46 +40,54 @@
...
+    public static final int ExitMsg              =14;
+    public static final int ReqStatMsg           =15;
+    public static final int StatMsg              =16;
+    public static final int StatDoneMsg          =17;
+    public static final int NodeBocInitMsg       =18;
+    public static final int ForNodeBocMsg        =19;
+    public static final int ArrayEltInitMsg      =20;
+    public static final int ForArrayEltMsg       =21;
+    public static final int ForIDedObjMsg        =22;
+    public static final int BocBcastMsg          =23;
```
Comment (rbuch) [inline charm/debug/pdata/MsgInfo.java:65]: Should this also deal with the lockless queue case?

Comment (epmikida) [reply charm/debug/pdata/MsgInfo.java:65]: I wasn't sure about that myself.

**Comment by evan-charmworks** [comment]
Should we merge this?

**Review by evan-charmworks — DISMISSED** [review]

**Comment by epmikida** [comment]
We might want to wait until after release? The bcast changes aren't part of charm 6.10 so they shouldn't be part of ccs tools 6.10.

**Comment by rbuch** [comment]
@epmikida Should this be updated and merged for 7.0.0? `MINOR` should now be 11 based on `debug-conv.h`.

**Comment by epmikida** [comment]
> @epmikida Should this be updated and merged for 7.0.0? `MINOR` should now be 11 based on `debug-conv.h`.

Yeah seems reasonable. I don't think there's any region not to merge it.

File: charm/debug/ParDebug.java [hunk 2]
```diff
@@ -39,7 +39,7 @@
      implements ActionListener,ListSelectionListener{
    
 	public final static int MAJOR = 10;
-	public final static int MINOR =  9;
+	public final static int MINOR = 10;
```
Comment (rbuch) [inline charm/debug/ParDebug.java]:
```suggestion
	public final static int MINOR = 11;
```

**Review by rbuch — CHANGES_REQUESTED** [review]

**Review by ericjbohm — DISMISSED** [review]

**Review by rbuch — APPROVED** [review]

**Pull request merged**